index.html

```diff
@@ -18,33 +18,34 @@
                 <p>You <em>know</em> you want to. </p>
                 <form>
                     <div class="shadow">
+                        <p>Let's do this!</p>
                     <div class="container3">
                         <fieldset action="" method="post">
-                            <label for="Fist_Name">First Name:</label>
+                            <label for="Fist_Name">FIRST NAME</label>
                             <input type="text" name="First Name" id="First_Name" required>
                         </fieldset>
                         <fieldset action="" method="post">
-                            <label for="Last_Name">Last Name</label>
+                            <label for="Last_Name">LAst NAME</label>
```

Comment:
Last name here wasn't quite fully capitalized, should be LAST NAME, not LAst NAME